whats-new/latest-releases/liverig-5.md

```diff
@@ -5,6 +5,16 @@ description: >-
 
 # LiveRig 5
 
+## 5.1 series
+
+### 5.1.1
+
+* [fixed] The widget has hidden his series on the first render
+
```

Comment:
```suggestion
```

we just inform new features/fixes at .0 versions (minor and majors), the details about bugfixes are provided only at marketplace.intelie.com